src/lib.rs

```diff
@@ -269,34 +270,39 @@ fn reduce_multiline_entries(lines: &mut Vec<LineEntry>) {
 fn find_multiline_ranges(lines: &[LineEntry]) -> Vec<(usize, usize)> {
     let mut multiline_ranges: Vec<(usize, usize)> = Vec::new();
     let mut start_number: Option<usize> = None;
+    let mut quote_char: Option<char> = None;
 
     // here we find ranges of multi-line values
     lines.iter().for_each(|entry| {
         if let Some(start) = start_number {
-            if let Some(idx) = entry.raw_string.find('\'') {
-                if !is_escaped(&entry.raw_string[..idx]) {
-                    multiline_ranges.push((start, entry.number));
+            if let Some(quote_char) = quote_char {
+                if let Some(idx) = entry.raw_string.find(quote_char) {
+                    if !is_escaped(&entry.raw_string[..idx]) {
+                        multiline_ranges.push((start, entry.number));
+                        start_number = None;
+                    }
+                } else if entry.get_value().is_some() {
```

Comment:
This was done in a previous commit (#450)
However, I am still afraid that it could have been done in another way.

I did this with backward compatibility to check for values starting with a quotation mark (not multi-line, see `QuoteCharacterChecker`). But what if multiline can contain char `=`?

We have to find the right way to distinguish between a valid `env` string from a multiline string with the `=` char